sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -1273,7 +1278,10 @@ case class GpuRegExpExtract(
       case i: Int if i == 0 =>
         ("(" + cudfRegexPattern + ")", 0)
       case i =>
-        (cudfRegexPattern, i.asInstanceOf[Int] - 1)
+        // (cudfRegexPattern, i.asInstanceOf[Int] - 1)
```

Comment:
Minor nit, not mustfix: Normally we don't leave commented code, the comment below should stand on its own.